cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -43,7 +43,7 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
       p(
         """In this document we will often use the terms _'allows'_ and _'enables'_ in seemingly identical ways. However, there is a subtle difference.
           |We will use _'enables'_ to refer to the consequences of <<administration-security-reads, read privileges>> where a restriction will not cause an error,
-          |only a reduction in the apparent graph size. We will use _'allows'_ to refer to the consequence of <<administration-security-write, write privileges>>
+          |only a reduction in the apparent graph size. We will use _'allows'_ to refer to the consequence of <<administration-security-writes, write privileges>>
```

Comment:
This link seems to be broken in 4.1 documentation as well: https://neo4j.com/docs/cypher-manual/4.1/administration/security/subgraph/#